www/js/main-layout.controller.js

```diff
@@ -20,6 +20,9 @@ angular
             currentAccount = accountService.getDefault();
             if (currentAccount && _.keys(currentAccount).length) {
                 if (currentAccount.country) {
+                    vm.showNetworkStatus = _.startsWith(currentAccount.loginid, 'MX') ||
+                      _.startsWith(currentAccount.loginid, 'MLT') ||
+                      _.startsWith(currentAccount.loginid, 'MF');
```

Comment:
you should check for landing company instead of hardcoding the broker code. please review the code and change where applicable